### PR TITLE
Update changelog for 0.33.0

### PR DESCRIPTION
**Pull request by sklam**
as titled

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+This release upgrades to LLVM 9 and drops support for older LLVM versions.
+
+Pull requests:
+
+* PR #593: Fix CUDA with LLVM9
+* PR #592: Fix meta.yaml
+* PR #591: buildscripts: Unpin wheel
+* PR #590: add python_requires to setup.py
+* PR #586: Update changelog for 0.32.1 on master
+* PR #585: Update changelog for 0.32.1 on release branch
```
Comment (stuartarchibald) [inline CHANGE_LOG:13]: Do these need to go in ? I removed similar from the Numba change log.

File: CHANGE_LOG [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+* PR #580: Trove classifiers may be out of date.
+* PR #582: Adds override for LLVM version check, re-formats docs.
+* PR #581: Add FAQ entry on LLVM version support.
+* PR #577: llvmlite wheel building fixes
+* PR #576: Update the release date
```
Comment (stuartarchibald) [inline CHANGE_LOG:18]: Is this on master?

File: CHANGE_LOG [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+
+Pull requests:
+
+* PR #593: Fix CUDA with LLVM9
+* PR #592: Fix meta.yaml
+* PR #591: buildscripts: Unpin wheel
+* PR #590: add python_requires to setup.py
+* PR #586: Update changelog for 0.32.1 on master
+* PR #585: Update changelog for 0.32.1 on release branch
+* PR #580: Trove classifiers may be out of date.
```
Comment (stuartarchibald) [inline CHANGE_LOG:14]: This should be 2 lines down, after #581

File: CHANGE_LOG (hunk 1 above)
Comment (sklam) [reply CHANGE_LOG:13]: right, they don't need to

File: CHANGE_LOG (hunk 2 above)
Comment (sklam) [reply CHANGE_LOG:18]: good catch.

File: CHANGE_LOG (hunk 3 above)
Comment (sklam) [reply CHANGE_LOG:12]: i guess my script is going by some sort of dates

File: CHANGE_LOG [hunk 4]
```diff
@@ -1,7 +1,30 @@
-v0.33.0dev
+v0.33.0rc1
```
Comment (stuartarchibald) [inline CHANGE_LOG:1]: I think this has the proposed release date in it now and no `rc1`?

**Pull request merged**